src/components/ListOfClasses.jsx

```diff
@@ -0,0 +1,21 @@
+export default function ListOfClasses(props) {
```

Comment:
This is good. You can also destruct props directly in your component's parameters like so:
```
export default function ListOfClasses({ classes }) {
  return (
    <>
      {classes.map(...)} instead of {props.classes.map(...)}
    </>
  )
}
```